src/Opulence/Framework/Databases/Console/Commands/FixMigrationsCommand.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+/*
+ * Opulence
+ *
+ * @link      https://www.opulencephp.com
+ * @copyright Copyright (C) 2017 David Young
+ * @license   https://github.com/opulencephp/Opulence/blob/master/LICENSE.md
+ */
+
+namespace Opulence\Framework\Databases\Console\Commands;
+
+use Exception;
+use Opulence\Console\Commands\Command;
+use Opulence\Console\Responses\IResponse;
+use Opulence\Databases\IConnection;
+use Opulence\QueryBuilders\MySql\QueryBuilder as MySqlQueryBuilder;
+use Opulence\QueryBuilders\PostgreSql\QueryBuilder as PostgreSqlQueryBuilder;
+use Opulence\QueryBuilders\QueryBuilder as BaseQueryBuilder;
+use RuntimeException;
+
+/**
+ * Defines the command that fixes migrations
+ */
+class FixMigrationsCommand extends Command
```

Comment:
Why is this a console command as opposed to another migration?

Reply:
I'm not even entirely sure what that would mean here. I could of course try to do something similar in `SqlExecutedMigrationRepository` like how check for the existence of the table works, but I think it would be properly ugly and also - less importantly - inefficient.

This command would be just run once on upgrade and then forgotten, hopefully.

Reply:
Yeah, after digging into the migrator code a bit more, I realized that things might be weird if the migration hasn't been run yet, but the migrator is depending on ordering by an ID column.